cocli/data/templates/comid-cca-realm-refval.json

```diff
@@ -0,0 +1,70 @@
+{
+    "lang": "en-GB",
+    "tag-identity": {
+        "id": "43BBE37F-2E61-4B33-AED3-53CFF1428B16",
+        "version": 0
+    },
+    "entities": [
+        {
+            "name": "Workload Client Ltd.",
+            "regid": "https://workloadclient.example",
+            "roles": [
+                "tagCreator",
+                "creator",
+                "maintainer"
+            ]
+        }
+    ],
+    "triples": {
+        "reference-values": [
+            {
+                "environment": {
+                    "class": {
+                        "id": {
+                            "type": "uuid",
+                            "value": "CD1F0E55-26F9-460D-B9D8-F7FDE171787C"
+                        },
+                        "vendor": "Workload Client Ltd"
+                    }
+                },
+                "measurements": [
+                    {
+                        "value": {
+                            "digests": [
+                                "sha-384;QoS1aUymwNLPR4mguVrIAlyBjeUjBDZL580pgbLS7caFsyInfsJYGZYkE9jJssH1"
+                            ]
+                        }
+                    },
+                    {
+                        "value": {
+                            "digests": [
+                                "sha-384;IQe752H8pS2VE2oTVNt6TdV7Gya+DT2nHZ6yOYazS6YVq/ZRTPNeWp6lWgMtBop4"
+                            ]
+                        }
+                    },
+                    {
+                        "value": {
+                            "digests": [
+                                "sha-384;QQe752H8pS2VE2oTVNt6TdV7Gya+DT2nHZ6yOYazS6YVq/ZRTPNeWp6lWgMtBop4"
+                            ]
+                        }
+                    },
+                    {
+                        "value": {
+                            "digests": [
+                                "sha-384;JQe752H8pS2VE2oTVNt6TdV7Gya+DT2nHZ6yOYazS6YVq/ZRTPNeWp6lWgMtBop4"
+                            ]
+                        }
+                    },
+                    {
+                        "value": {
+                            "digests": [
+                                "sha-384;MQe752H8pS2VE2oTVNt6TdV7Gya+DT2nHZ6yOYazS6YVq/ZRTPNeWp6lWgMtBop4"
+                            ]
```

Comment:
two observations:

1. CoRIM only allows one measurement-map in a reference-triple-record 
2. these measurements are not named, so it's not clear what they refer to (I assume there is some sort of positional convention to distinguish RIM from REMs?)

Reply:
Observation - 1: Correct: However we need to change that in the base CoRIM Schema.

All these Measurements belong to one `Environment` so really has to be in One Measurement Map

2. Yes, the positional convention is implicit, The first value will always be RIM Value and the subsequent will be REM's
3. We can add a comment in the `Supply Chain Documentation` that the first measurement always has to be `RIM`

Reply:
> Observation - 1: Correct: However we need to change that in the base CoRIM Schema.

That is not going to happen.  There is too much resistance to that.

Reply:
> > Observation - 1: Correct: However we need to change that in the base CoRIM Schema.
> 
> That is not going to happen. There is too much resistance to that.

I plan to re-initiate the discussion: